project/dexter-vs/Dexter/Common/Core/Client/DexterClient.cs

```diff
@@ -79,6 +81,21 @@ public Task StoreSourceCodeCharSequence(long snapshotId, long groupId, string mo
             }
         }
 
+        public async Task AddAccount(string userName, string userPassword, bool isAdmin)
+        {
+            var queryParams = HttpUtility.ParseQueryString("");
+            queryParams.Add("userId", userName);
+            queryParams.Add("userId2", userPassword);
+            queryParams.Add("isAdmin", isAdmin.ToString());
+
+            HttpResponseMessage response = await httpClient.PostAsync($"{POST_ACCOUNT_ADD_V1}?{queryParams}","");
+
+            if (!response.IsSuccessStatusCode.Equals(true))
+            {
+                Debug.WriteLine(response, "Failed to add account!");
```

Comment:
The error is being ignored. It would be better to return the success / failure of account creation.